src/main/java/org/folio/circulation/resources/AbstractCirculationRulesEngineResource.java

```diff
@@ -27,22 +25,21 @@
  * The circulation rules engine calculates the loan policy based on
  * item type, loan type, patron type and shelving location.
  */
-public class CirculationRulesEngineResource extends Resource {
-  private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+public abstract class AbstractCirculationRulesEngineResource extends Resource {
```

Comment:
I reckon we might be able to convert this abstract class into a parameterised class that can be used for all three kinds of policies. However I think that refactoring can wait till another day.